### DISCUSSION TITLE
Dependency Dashboard issue is created every time I run Renovate

### DISCUSSION THREAD

**Discussion by olegkrivtsov** [renovatebot/renovate#12219] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Branch master commit 2a09db0f23fd7b6d792ad3db168b3b260edede9e

### Describe the bug

Every time I run `yarn start` against my test repo https://github.com/olegkrivtsov/renovate-rebase-test, Renovate creates a Dependency Dashboard issue. Now I have 11 issues with the same name. I'm not sure whether this problem is on my side or this is a Renovate bug.

My git version is 2.33.1

### Relevant debug logs

<details><summary>Logs</summary>

```
{"name":"renovate","hostname":"oleg-work","pid":51051,"level":20,"logContext":"2cek9HOv3","repository":"olegkrivtsov/renovate-rebase-test","msg":"Ensuring Dependency Dashboard","time":"2021-10-19T11:08:32.610Z","v":0}
{"name":"renovate","hostname":"oleg-work","pid":51051,"level":20,"logContext":"2cek9HOv3","repository":"olegkrivtsov/renovate-rebase-test","msg":"ensureIssue(Dependency Dashboard)","time":"2021-10-19T11:08:32.611Z","v":0}
{"name":"renovate","hostname":"oleg-work","pid":51051,"level":30,"logContext":"2cek9HOv3","repository":"olegkrivtsov/renovate-rebase-test","msg":"Issue created","time":"2021-10-19T11:08:33.325Z","v":0}
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment]
You need to debug it

**Comment by viceice** [comment]
You probably have a mismatch between the PAT and the configured username, so renovate can't find its own issues

**Comment by olegkrivtsov** [comment, marked as answer]
Hi @viceice thank you, probably that was the case. I edited my `config.js` as follows and it worked:

```js
module.exports = {
  token: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
  platform: 'github',
  onboardingConfig: {
    extends: ['config:base'],
  },
  repositories: ['olegkrivtsov/renovate-rebase-test'],
};
```

This issue was on my side.
